.github/workflows/rhobs-release.yaml

```diff
@@ -60,7 +60,9 @@ jobs:
           TAG="v$(head -1 VERSION)" \
           ./rhobs/push-container-images.sh
 
-          echo ::set-output name=tag_name::$(git describe HEAD --abbrev=0)
+          cd rhobs/olm
+          make tools
+          make bundle-image bundle-push IMAGE_BASE=quay.io/$IMAGE_ORG/rhobs-prometheus-operator
```

Comment:
This is deliberate, perhaps we should rename everything to rhobs instead of `obo`. The operator where going to submit to Community catalog  is `RHOBS Prometheus Operator` and hence the bundle uses `rhobs-`. The original intent of this operator was to only serve as a dependency of `obo` and hence all operator related images are `obo`. 
I will try and fix this in subsequent release to avoid the confusion.

Reply:
On second thought, I will just use `obo` everywhere